README.md

```diff
@@ -47,7 +47,7 @@ This portlet requires a qPortal environment, which has as a prerequisite the fol
 
 1. A portal running on [Liferay](https://www.liferay.com/de/home). Our portal has been tested on [Liferay 6.2 bundled with Tomcat](http://www.liferay.com/downloads/liferay-portal/available-releases).
 2. An [OpenBIS](https://openbis.ch/) server (biological data management system). 
-3. An OMERO server instance, this software is currently compatible with version [5.4.10](https://docs.openmicroscopy.org/omero/5.4.10/). 
+3. An OMERO server instance, this software is currently compatible with version [5.6.7](https://omero.readthedocs.io/en/v5.6.7/) 
```

Comment:
This seems to be not the latest link: 
Read the docs recommends the following link:

https://omero.readthedocs.io/en/v5.6.7-4/

```suggestion
3. An OMERO server instance, this software is currently compatible with version [5.6.7](https://omero.readthedocs.io/en/v5.6.7/) 
```

Reply:
Thanks, I'll point to the recommended doc